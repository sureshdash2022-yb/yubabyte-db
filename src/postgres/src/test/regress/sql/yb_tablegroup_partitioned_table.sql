CREATE DATABASE test_tablegroup_partitioned_tables;
\c test_tablegroup_partitioned_tables
CREATE TABLEGROUP tg1;
CREATE TABLE prt (id int PRIMARY KEY, v int) PARTITION BY RANGE (id) TABLEGROUP tg1;
CREATE TABLE prt_p1 PARTITION OF prt FOR VALUES FROM (1) TO (2);
CREATE TABLE IF NOT EXISTS prt_p2 PARTITION OF prt FOR VALUES FROM (2) TO (3);
\dgrt
CREATE TABLE prt_p3 PARTITION OF prt FOR VALUES FROM (3) TO (4) TABLEGROUP tg1;
CREATE TABLE IF NOT EXISTS prt_p4 PARTITION OF prt FOR VALUES FROM (4) TO (5) TABLEGROUP tg1;
