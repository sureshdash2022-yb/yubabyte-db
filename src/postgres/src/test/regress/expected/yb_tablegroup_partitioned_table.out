CREATE DATABASE test_tablegroup_partitioned_tables;
\c test_tablegroup_partitioned_tables
CREATE TABLEGROUP tg1;
CREATE TABLE prt (id int PRIMARY KEY, v int) PARTITION BY RANGE (id) TABLEGROUP tg1;
CREATE TABLE prt_p1 PARTITION OF prt FOR VALUES FROM (1) TO (2);
CREATE TABLE IF NOT EXISTS prt_p2 PARTITION OF prt FOR VALUES FROM (2) TO (3);
\dgrt
              List of tablegroup tables
 Group Name | Group Owner |  Name  | Type  |  Owner   
------------+-------------+--------+-------+----------
 tg1        | yugabyte    | prt    | table | yugabyte
 tg1        | yugabyte    | prt_p1 | table | yugabyte
 tg1        | yugabyte    | prt_p2 | table | yugabyte
(3 rows)

CREATE TABLE prt_p3 PARTITION OF prt FOR VALUES FROM (3) TO (4) TABLEGROUP tg1;
ERROR:  syntax error at or near "TABLEGROUP"
LINE 1: ...rt_p3 PARTITION OF prt FOR VALUES FROM (3) TO (4) TABLEGROUP...
                                                             ^
CREATE TABLE IF NOT EXISTS prt_p4 PARTITION OF prt FOR VALUES FROM (4) TO (5) TABLEGROUP tg1;
ERROR:  syntax error at or near "TABLEGROUP"
LINE 1: ...rt_p4 PARTITION OF prt FOR VALUES FROM (4) TO (5) TABLEGROUP...
                                                             ^
